File: 04_Sorting/selection.sort.js
let arr = [5, 3, 1, 2, 4];

 (
    function selectionSort(arr){
        let min = 0 
        for(let i = 0 ; i < arr.length ; i++){
            min = i
            for(let j = i ; j < arr.length ; j++){

                if(arr[j]<arr[min]){
                    min = j
                }

            }

            let temp = arr[i]
            arr[i] = arr[min]
            arr[min] = temp

        }
    }
 )(arr)

 console.log(arr);

File: java/sorting/src/SelectionSort.java
import java.util.Arrays;

public class SelectionSort {
    public static void main(String[] args) {
        int[] arr = {5,4,3,2,1};

        for (int i = 0; i < arr.length ; i++) {
            int litem = 0;
            for (int j = 1; j < arr.length - i; j++) {
                if (arr[litem] < arr[j]){
                    litem = j;
                }
            }
            int temp = arr[litem];
            arr[litem] = arr[arr.length - 1  - i];
            arr[arr.length - 1  - i] = temp;
        }

        System.out.println(Arrays.toString(arr));
    }

}
